samples/msal-angular-samples/MSALAngularDemoApp/.beachballrc

```diff
@@ -0,0 +1,3 @@
+{
+    "shouldPublish": false
```

Comment:
Is there a way to ignore the samples folder completely so we don't have to add this to every sample?

Reply:
I wanted to use the path but beachball is currently overwriting the path config parameter with `cwd` so until they fix their config we need to do this.

Reply:
Got it. Makes sense

Reply:
I opened a ticket here: https://github.com/microsoft/beachball/issues/377